.pre-commit-config.yaml

```diff
@@ -0,0 +1,13 @@
+# See https://pre-commit.com for more information
+# See https://pre-commit.com/hooks.html for more hooks
+
+repos:
```

Comment:
TODO: Should we add `pre-commit` to requirements.txt or something? Not sure how this repo handles dev dependencies, a bit confusing...